### PR TITLE
Lee alogrithm branch

### PR DESCRIPTION
**Pull request by nicusor43**

### CODE REVIEW

File: Maze Routing Algorithms/lee_alogrithm.cpp [hunk 1]
```diff
@@ -0,0 +1,58 @@
+#include <queue>
+#include <iostream>
+
```
Comment (siddharth25pandey) [inline Maze Routing Algorithms/lee_alogrithm.cpp:3]: @nicusor43 Better to use a Single Header file **bits/stdc++**

File: Maze Routing Algorithms/lee_alogrithm.cpp [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+// These constants will be the size of our 2d array
+const int n = 5, m = 5;
+
+// Our 2d array
+int vec[n][m] = {{ 0,  0, 0, -1, 0},
+                 {-1, -1, 0, -1, 0},
+                 {-1, 0,  0, -1, 0},
+                 {-1, 0, -1, 0, -1},
+                 {-1, 0,  0, 0,  0}};
+
```
Comment (siddharth25pandey) [inline Maze Routing Algorithms/lee_alogrithm.cpp:20]: Take Dynamic Input from the user for more better friendly user interface

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@nicusor43 Do the suggested changes
Thanks!

**Comment by nicusor43** [comment]
I made the changes

**Review by siddharth25pandey — APPROVED** [review]
@nicusor43 Good to go
Able to merge
Thanks!

**Pull request merged**